synapse/cortex.py

```diff
@@ -890,6 +890,7 @@ async def finidmon():
         import synapse.lib.spawn as s_spawn  # get around circular dependency
         self.spawnpool = await s_spawn.SpawnPool.anit(self)
         self.onfini(self.spawnpool)
+        self.spawncorector = None
```

Comment:
It's that we can store the ctor here `self.spawncorector = f'{s_spawn.__name__}.{s_spawn.SpawnCore.__name__}'`
but that make it a bit ickier to subclass the Cortex.

```class SubCortex(s_cortex.Cortex):

    async def __anit__(self, dirn, conf=None):
        await s_cortex.Cortex.__anit__(self, dirn, conf=conf)
        self.spawncorector = f'{s_some_spawn.__name__}.{s_some_spawn.SomeSpawnCore.__name__}'
```

feels icky

Reply:
See my comment below ( we probably shouldn't store it at all )